public/css/style.css

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+  text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body{
+  font-family: 'Droid Serif', serif;
+  font-size: 14px;
+  color: #2f2f2f;
+  background-color: #f9f7f1;
+}
+
+.page-container {
+  min-height: 98vh;
+  position: relative;
+}
+
+.content-wrap {
+  padding-bottom: 2.5rem;
+}
+
+footer {
+  position: absolute;
+  bottom: 0;
+  height: 2.5rem;
+  text-align: center;
+  width: 100%;
+  padding-top: 40px;
+  padding-bottom: 40px;
+  margin-top: 40px;
+}
+
+
+.headline {
+  font-size: 20px;
+  font-weight: 900;
+}
+
+@media (min-width: 1000px) {
```

Comment:
assuming this is to do with phone display? would love to know how tricky this was. I loaded on my phone and I couldn't see any of the headlines but I was also getting a warning saying the connection wasn't secure, so I think my phone might have been blocking the api requests. Anyway, it fit the display perfectly!

Reply:
yah, something I found online. Interesting 🤔 I think I needed to deploy it to an http address, maybe that would have fixed the security thing. Thanks for the feedback 🙌